src/app/featureSets/basic/render2D.js

```diff
@@ -525,6 +525,33 @@ export const render2D = {
         targetPrimitiveType: "MixerTarget",
         targetPrimitiveSet: "Basic2D"
     },
+    ToroidalMixer: {
+        featureParams: {
+            componentSpacing: "componentSpacing",
+            position: "position",
+            channelWidth: "channelWidth",
+            neckAngle: "neckAngle",
+            neckLength: "neckLength",
+            neckWidth: "neckWidth",
+            numberOfMixers: "numberOfMixers",
+            rotation: "rotation",
+            innerDiameter: "innerDiameter"
```

Comment:
Is this still being used ? AFAIK, this is no longer relevant with the new component API we have